extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php

```diff
@@ -189,6 +189,14 @@
 						<li><?= wfMessage( 'user-identity-box-zero-state-gender' )->escaped(); ?></li>
 					<? endif; ?>
 				<? endif; ?>
+				<? if ( !empty( $user['bio'] ) ): ?>
+					<li id="discussion-bio"><?= wfMessage( 'user-identity-bio', $user['bio'] )->plain(); ?></li>
```

Comment:
This is vulnerable to stored XSS since the user can put whatever they want in the bio attribute in the service.